### ISSUE TITLE
Skjul ikoner fra skjermlesere og endre koding på brukergrupper (Privatperson osv).

### ISSUE THREAD

**Issue by anders-nom** [navikt/nav-enonicxp-frontend#668]
Skjermlesere leser opp title-tagger fra svg-filer som brukes som ikoner i lenkepaneler. Å skjule ikonene med aria-hidden vil kanskje fikse problemet?

Brukergrupper er kodet med tablist/tab,
Det er foreslått å ta vekk aria-attributter og bruke aria-current=’page’ for å vise valgt målgruppe.

I lesemodus (Jaws) lese opp Privatperson, Arbeidsgiver osv. dobbelt i dag. Ref til diskusjon i Slack: https://nav-it.slack.com/archives/CAFRFDJMN/p1641812615012100
